assets/src/scss/blocks/SecondaryNavigation/SecondaryNavigationStyle.scss

```diff
@@ -0,0 +1,322 @@
+.secondary-navigation-block {
+  margin-left: calc(-50vw - -50%);
+  width: 100vw;
+  height: 65px;
+  padding: 10px;
+  background: var(--p4-dark-green-800);
+  position: relative;
+
+  @include medium-and-less {
+    > .container {
+      max-width: 100%;
+    }
+
+    position: absolute;
+    left: 0;
+    height: 0;
+    padding: 0;
+    overflow: hidden;
+    transition: height 0.3s linear;
+  }
+
+  .EmptyMessage {
+    text-align: center;
```

Comment:
Is this something that we'd want to add for all `EmptyMessage` components?

Reply:
I can't say for sure, seems to work well in the Guestbook block but then again that is used in a p tag